test/test_functions

```diff
@@ -1140,6 +1140,18 @@ check_repository() {
   return 0
 }
 
+check_repo_integrity() {
+  local repo=$1
+  for f in `find /cvmfs/${repo} -type f`; do
```

Comment:
More compact:

    if find /cvmfs/${repo} -type f -exec cat {} >/dev/null \; ; then

Reply:
I wanted to print out the corrupted file, which would be messy with the compact version. Is this really necessary?